collectors/ebpf.plugin/ebpf.c

```diff
@@ -1695,9 +1703,13 @@ static void read_collector_values(int *disable_apps)
         started++;
     }
 
-    enabled = appconfig_get_boolean(&collector_config, EBPF_PROGRAMS_SECTION, "network connection monitoring",
-                                    0);
-    ebpf_modules[1].optional = enabled;
+    // This is kept to keep compatibility
+    value = appconfig_get(&collector_config, EBPF_PROGRAMS_SECTION, "network connection monitoring",
+                                    NULL);
+    if (!value)
+        value = appconfig_get(&collector_config, EBPF_PROGRAMS_SECTION, "network connection",
+                              "no");
+    ebpf_modules[1].optional = (!value || !strcasecmp(value, "no"))?0:1;
```

Comment:
Use `appconfig_get_boolean` and `CONFIG_BOOLEAN_YES`/`CONFIG_BOOLEAN_NO`.

Reply:
I used text, because it was only parsing the config file, but I agree that we can use `get_boolean`. I changed the PR with the last commits.